index.css

```
@@ -0,0 +1,311 @@
+
+:root {
+  --linkbrary-primary-color: #6D6AFE;
+  --linkbrary-red: #FF5B56;
+  --the-julge-black: #111322;
+  --linkbrary-white: #FFFFFF;
+  --linkbrary-gray-100: #3E3E43;
+  --linkbrary-gray-60: #9FA6B2;
+  --linkbrary-gray-20: #CCD5E3;
+  --linkbrary-gray-10: #E7EFFB;
+  --linkbrary-bg: #F0F6FF;
+  --grey-light: #F5F5F5;
+  --gra-purpleblue-to-skyblue: linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
+  --fandom-k-gra-blue-to-pink: linear-gradient(271deg, #FE578F -9.84%, #68E8F9 107.18%)
+}
+
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Pretendard, Abel, sans-serif;
+  word-break: keep-all;
+}
+
+body {
+  max-width: 1920px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  /*gap: 8px;*/
+  background: #FFF;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+header {
+  z-index: 1;
+  position: sticky;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
+  align-self: stretch;
+  background: var(--linkbrary-bg, #F0F6FF);
+  padding: 20px 200px;
+}
+
+header nav {
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+}
+
+a[href] {
+  text-decoration: none;
+}
+
+.logo {
+  width: 133px;
+  height: 24px;
+}
+
+#login {
+  display: flex;
+  width: 128px;
+  padding: 16px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  border-radius: 8px;
+  background: var(--gra-purpleblue-to-skyblue);
```

저는 backgroung-color를 줄때는 background보다는 더 명확한 요소로 주는 편이에요. 왜냐하면 background는 color말고도 다른 요소가 있어서 해당 프로퍼티를 읽을떄는 이 요소엔 왜 color만 있지라는 의문이 생기더라구요,
```suggestion
  background-color: var(--gra-purpleblue-to-skyblue);
```

---

주신 코멘트대로 색상을 background 대신, background-color로 대체해서 적용해보려했는데 안 되더라구요! 

조사해보니 ,linear-gradient() 함수는 두 개 이상의 색이 직선을 따라 점진적으로 변화하는 이미지를 생성한다고합니다.  -> 적용하려는 배경색상이 linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%)를 변수화한 컬러여서 색상속성으로는 적용이 불가했습니다.

---

아하 확인했습니다! gradient이외에는 backgroundColor로 부탁드릴게요~